api/nakadi-event-bus-api.yaml

```diff
@@ -1567,13 +1648,14 @@ definitions:
           future there could be others.
       schema:
         type: string
-        $ref: '#/definitions/EventTypeSchema'
         description: |
           The schema as string in the syntax defined in the field type. Failure to respect the
           syntax will fail any operation on an EventType.
 
-          To have a generic, undefined schema it is possible to define the schema as `"schema":
-          "{\"additionalProperties\": true}"`.
```

Comment:
Unless it is restricted to create new event types with "legacy" compatible mode.